.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: native-metrics CI
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
```

Comment:
Looks like this will run/deploy on branch pushes with the v* tag. 

So after we've merged release note PR's, and done an `npm version` but prior to doing a manual npm deploy.

This is all pretty consistent with how the previous was setup.